### PR TITLE
cli/commands: Add desc as an alias for description

### PR DESCRIPTION
**Pull request by honglooker**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Review by martinvonz — APPROVED** [review]
This is fine with me. It doesn't seem like it would conflict with any future commands, and I don't think aliases make any difference when it comes to shell completion (at least not for Fish). Let's give others a chance to comment, though.

File: cli/src/commands/mod.rs
Comment (PhilipMetzger) [inline cli/src/commands/mod.rs:1]: Nit: the commit title should be prefixed with `cli/commands:` or `commands:` to match the repo's convention.

**Review by PhilipMetzger — APPROVED** [review]
LG, I have no issue with this alias.

File: cli/src/commands/mod.rs
Comment (honglooker) [reply cli/src/commands/mod.rs:1]: sure - done

**Comment by martinvonz** [comment]
> LG, I have no issue with this alias.

Alright, that's good enough for me. Feel free to merge, @honglooker. Worst case we roll it back :)

File: cli/src/commands/mod.rs
Comment (PhilipMetzger) [reply cli/src/commands/mod.rs:1]: you changed the pr's title not the commit message, which I wanted. So ideally repush the commit with the commit message change.

Comment (honglooker) [reply cli/src/commands/mod.rs:1]: Ah, thanks for clarifying. I'll be more mindful of this in future PRs

Comment (martinvonz) [reply cli/src/commands/mod.rs:1]: I don't think we have a single commit with a "cli/commands:" prefix so far, so I don't think it's much a convention anyway. FWIW, I often use a "cli:" prefix, but  I don't really care as long as it's clear (maybe "Add desc as an alias for for the description *command*" would have helped, btw, so it's clear that it's not the "description" template it's about). Some contributors follow https://www.conventionalcommits.org/

**Pull request merged**